src/test/java/uk/gov/companieshouse/api/accounts/validation/AccountsValidatorTest.java

```diff
@@ -102,7 +141,7 @@ private void setUp() throws DataException {
     @DisplayName("Validate Submission - successful, no errors")
     void validateSubmissionNoErrorsFound() throws DataException {
 
-        Errors errors = new Errors();
+        Errors errors = new Errors(); //Empty.
```

Comment:
You could call this emptyErrors instead of errors and get rid of the comment. Then it's clear throughout the code.